tests/integration/test_parquet_page_index/test.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+from helpers.cluster import ClickHouseCluster
+import pyarrow.parquet as pq
+import os
+import time
+
+cluster = ClickHouseCluster(__file__)
+path_to_userfiles = "/var/lib/clickhouse/user_files/"
+path_to_external_dirs = (
+    "/ClickHouse/tests/integration/test_parquet_page_index/_instances"
```

Comment:
I believe there was a better way to obtain path to `_instances` dir. Something like `cluster.instances_dir_name`. But anyway, I don't understand why is it needed. `external_dirs` argument in `cluster.add_instance` function maps this path as absolute in container, but makes it relative to `_instances` dir on host

Reply:
yes, cluster.instances_dir.
I need this to open the file from `/var/lib/clickhouse/user_files/` later. It is easer to open it on the test host rather then on cluster node, so it is easier to mount `/var/lib/clickhouse/user_files/`